[Bug]: A typo in the action.yml

### CheckBox

- [x] I have searched the issue list and found no similar issues.
- [x] I use the official action instead of someone else's
- [x] I have read the FAQ section in README.md

### Configuration

name: Kernel Build CI
on:
  workflow_dispatch:

jobs:
  build-kernel:
    runs-on: ubuntu-22.04

    steps:
      - uses: actions/checkout@v4

      - name: Build Kernel
        uses: dabao1955/kernel_build_action@main
        with:
          kernel-url: https://github.com/AsrieltheGoat/HeroGraceKernel8890
          kernel-branch: dev
          config: .config
          arch: arm64
          android-version: 10
          other-gcc64-url: https://github.com/LineageOS/android_prebuilts_gcc_linux-x86_aarch64_aarch64-linux-android-4.9
          other-gcc64-branch: lineage-19.1
          other-gcc32-url: https://github.com/LineageOS/android_prebuilts_gcc_linux-x86_arm_arm-linux-androideabi-4.9
          other-gcc32-branch: lineage-19.1
          disable-lto: true
          anykernel3: false
          bootimg-url: https://github.com/AsrieltheGoat/HeroGraceKernel8890/releases/download/hero2lte/HeroGraceKernel-V1.0-G935X-20251115-ksu.img


### Describe the bug

There is a typo in the action.yml script that caused the apt failed to install 

"E: Unable to locate package binutils-arm-linux-gnuabeihf"

### Screenshots

<img width="1540" height="703" alt="Image" src="https://github.com/user-attachments/assets/749b2000-f5f2-47f8-9fd9-e98efb5ff228" />

### How to reproduce

_No response_

### Expected behavior

_No response_

### Error Logs

_No response_

### Additional context

_No response_

> > Strange, this package is not among the installed dependencies. Needs further investigation.
> 
> [@devnoname120](https://github.com/devnoname120) Currently the patch encounters this error. Could the author help fix it? Thank you. ../security/selinux/hooks.c:2316:13: error: ISO C90 forbids mixing declarations and code [-Werror,-Wdeclaration-after-statement] static u32 ksu_sid; ^ 1 error generated. make[4]: *** [../scripts/Makefile.build:339: security/selinux/hooks.o] Error 1 make[3]: *** [../scripts/Makefile.build:653: security/selinux] Error 2 make[2]: *** [../Makefile:1143: security] Error 2 make[2]: *** Waiting for unfinished jobs....

@fyj69 It’s unrelated to my patch, ask whoever wrote this `hooks.c` file to fix it.

> > > Strange, this package is not among the installed dependencies. Needs further investigation.
> > 
> > 
> > [@devnoname120](https://github.com/devnoname120) Currently the patch encounters this error. Could the author help fix it? Thank you. ../security/selinux/hooks.c:2316:13: error: ISO C90 forbids mixing declarations and code [-Werror,-Wdeclaration-after-statement] static u32 ksu_sid; ^ 1 error generated. make[4]: *** [../scripts/Makefile.build:339: security/selinux/hooks.o] Error 1 make[3]: *** [../scripts/Makefile.build:653: security/selinux] Error 2 make[2]: *** [../Makefile:1143: security] Error 2 make[2]: *** Waiting for unfinished jobs....
> 
> [@fyj69](https://github.com/fyj69) It’s unrelated to my patch, ask whoever wrote this `hooks.c` file to fix it.

Thanks for the reply. I have already compiled the kernel successfully using the parameter KBUILD_CFLAGS=-Wno-declaration-after-statement.

--- a/action.yml
+++ b/action.yml
@@ -217,7 +217,7 @@ runs:
             fi
             echo "::endgroup::"
         else
-            SU apt-get install -y clang lld binutils-aarch64-linux-gnu binutils-arm-linux-gnuabeihf
+            SU apt-get install -y clang lld binutils-aarch64-linux-gnu binutils-arm-linux-gnueabihf
         fi
 
         if [ "${{ inputs.aosp-gcc }}" == "true" ] || [ -n "${NEED_GCC:-}" ]; then

